scipy/optimize/_shgo.py

```diff
@@ -305,58 +303,58 @@ def shgo(func, bounds, args=(), constraints=None, n=100, iters=1, callback=None,
     ...
     >>> bounds = [(-512, 512), (-512, 512)]
 
-    `shgo` has two built-in low discrepancy sampling sequences. First, we will
-    input 30 initial sampling points of the Sobol sequence:
+    `shgo` has built-in low discrepancy sampling sequences. First, we will
+    input 64 initial sampling points of the Sobol' sequence:
```

Comment:
no need for the apostrophe there.

Reply:
This is the name of the method which has an apostrophe `Sobol'`. I added some italic to help the user.

Reply:
I think it should either be:

> of Sobol's sequence

or

> of the Sobol sequence

but definitely not `Sobol'`